scripts/src/ui/page/multichoice.question.tsx

```diff
@@ -1,29 +1,53 @@
 import * as React from "react";
 import BasePage from "./base";
 import "../../../../styles/src/page/multichoice.question.styl";
+import { ButtonHTMLAttributes } from "react";
 
 export interface FullPageMultiChoiceProps {
 	id: string;
 	title: string;
+	description: string;
 	choices: string[];
-	onSelected?: (answerData: {}) => void;
+	onSelected: (answerData: {}) => void;
 }
 
-class MultichoiceQuestion extends React.Component<any, FullPageMultiChoiceProps> {
-	public render() {
-		const choices = this.props.choices.map((value: string, index: number) => {
-			return <button key={index} className="choice" onClick={() => {
+export interface FullPageMultiChoiceState {
+	disabled: boolean;
+}
+
+class MultichoiceQuestion extends React.Component<FullPageMultiChoiceProps, FullPageMultiChoiceState> {
+	private buttons: JSX.Element[] = [];
```

Comment:
If you're using a constructor, then I think that it's better to do all initialization inside of it.